sources/packages/web/src/views/aest/student/applicationDetails/AssessmentsSummary.vue

```diff
@@ -9,49 +9,52 @@
         }"
         subTitle="Assessments"
       />
```

Comment:
This page has more routes like NOA summary and NOA and view change request forms. Please make sure that the components under those routes are taken care as welll.